File: src/main/webapp/web/js/service/areasService.js
//服务层
app.service('areasService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../areas/findAll.do');		
	}
	
	//读取列表数据绑定到表单中
	this.findAreas=function(cityid){
		return $http.get('../areas/findAreas.do?cityid='+cityid);		
	}
});

File: src/main/java/com/pinyougou/user/services/AreasService.java
package com.pinyougou.user.services;
import java.util.List;
import com.pinyougou.pojo.TbAreas;
import com.pinyougou.pojo.TbCities;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface AreasService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbAreas> findAll();
	
	public List<TbAreas> findByAreas(String cityid);
	
}
